brave/src/main/java/brave/internal/Platform.java

```diff
@@ -163,12 +167,16 @@ public Clock clock() {
   }
 
   static class Jre9 extends Jre7 {
+    @Override public long currentTimeMicroseconds() {
```

Comment:
moved the actual (internal) function up so that we can use it and mock access to it easier.